components/Search/Searchbar.tsx

```diff
@@ -110,6 +131,36 @@ export default function Searchbar() {
                }).filter(x => !!x).slice(0, 5)}
             </>
             : null}
+         {titles && searchValue.length > 0 ?
+            <>
+               {titles.map((title) => {
+                  if (!title.name.toLowerCase().startsWith(searchValue)) {
+                     return null;
+                  }
+
+                  if (title.challengeId === undefined) { // TODO titles with no challange
```

Comment:
this whole bracket can be removed, it is duplicate / no response should be default and therefore fallback here is not needed

Reply:
Fixed in https://github.com/SariusGH/challenges/commit/f0be20bfbbdf41da709d39b03e684160292ab184